hera_qm/utils.py

```diff
@@ -630,7 +633,8 @@ def strip_extension(path, return_ext=False):
 
 
 def apply_yaml_flags(uv, a_priori_flag_yaml, lat_lon_alt_degrees=None, telescope_name=None,
-                     ant_indices_only=False, by_ant_pol=False, ant_pols=None):
+                     ant_indices_only=False, by_ant_pol=False, ant_pols=None,
```

Comment:
I'd like to see the unit tests of this function updated to include these new options.